### PR TITLE
add vue-router test

### PR DESCRIPTION
**Pull request by thomasleveil**
See #31

### CODE REVIEW

File: test/data/vue-router/vue-loader/components/app.vue [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<template lang="pug">
```
Comment (LavrovArtem) [inline test/data/vue-router/vue-loader/components/app.vue]: Please remove unnecessary folder `test/data/vue-router/vue-loader`

File: test/vue-router.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
+import VueSelector from '../lib';
+
+fixture `vue-loader`
```
Comment (LavrovArtem) [inline test/vue-router.js]:
```js
fixture `vue-router`
```

File: test/vue-router.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+const routerLinks = VueSelector('router-link');
+
+test('select component in main app', async t => {
+    await t
+        .expect(routerLinks.count).eql(2);
+});
+
+
+test('vue-router works', async t => {
```
Comment (miherlosev) [inline test/vue-router.js]: Rename test to `selector`

File: test/vue-router.js [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+fixture `vue-loader`
+    .page `http://localhost:8080/test/data/vue-router/`;
+
+
+const routerLinks = VueSelector('router-link');
+
+test('select component in main app', async t => {
+    await t
+        .expect(routerLinks.count).eql(2);
```
Comment (miherlosev) [inline test/vue-router.js]: Move this expection to the second test and remove test

File: test/vue-router.js [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+        .click(routerLinks.nth(0))
+        .expect(VueSelector('foo').exists).ok()
+        .expect(VueSelector('bar').exists).notOk()
+        .click(routerLinks.nth(1))
+        .expect(VueSelector('foo').exists).notOk()
+        .expect(VueSelector('bar').exists).ok();
+});
+
+
+test('select component in foo', async t => {
```
Comment (miherlosev) [inline test/vue-router.js]: Rename test to `component properties`

File: test/vue-router.js [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+    const listItemVue2   = await listItem.nth(1).getVue();
+    const listItemVue3Id = listItem.nth(2).getVue(({ props }) => props.id);
+
+    await t
+        .expect(listItem.count).eql(3)
+        .expect(listItemVue2.props.id).eql('list-foo-1-item2')
+        .expect(listItemVue3Id).eql('list-foo-1-item3');
+});
+
+test('select component in bar', async t => {
```
Comment (miherlosev) [inline test/vue-router.js]: Remove this test, because you already check component selection and component properties in previous two tests.

**Comment by thomasleveil** [comment]
Here you go

**Pull request merged**